.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/AssemblyInfoVersion"]
```

Comment:
can we have it at the root of the repo, instead of having it in /src ? 

I'd like to keep it separated, and have all submodules available from the root of the repo.